### PR TITLE
Support for x argument for upgrade and downgrade

### PR DESCRIPTION
**Pull request by jessepavlis**
After reading the [Conditional Migration Elements](http://alembic.readthedocs.org/en/latest/cookbook.html#conditional-migration-elements) section for alembic and Issue https://github.com/miguelgrinberg/Flask-Migrate/issues/32 I thought it would be nice to have the x argument available to be used direct with Flask-Migrate.

### CODE REVIEW

File: flask_migrate/__init__.py [hunk 1]
```diff
@@ -172,9 +179,12 @@ def merge(directory=None, revisions='', message=None, branch_label=None,
 @MigrateCommand.option('-d', '--directory', dest='directory', default=None,
                        help=("migration script directory (default is "
                              "'migrations')"))
-def upgrade(directory=None, revision='head', sql=False, tag=None):
+@MigrateCommand.option('-x', '--x-arg', dest='x_arg', default=None,
+                       help=("Additional arguements consumed by "
```
Comment (miguelgrinberg) [inline flask_migrate/__init__.py]: Typo in the help message: arguements --> arguments

File: flask_migrate/__init__.py [hunk 2]
```diff
@@ -189,9 +199,12 @@ def upgrade(directory=None, revision='head', sql=False, tag=None):
 @MigrateCommand.option('-d', '--directory', dest='directory', default=None,
                        help=("migration script directory (default is "
                              "'migrations')"))
-def downgrade(directory=None, revision='-1', sql=False, tag=None):
+@MigrateCommand.option('-x', '--x-arg', dest='x_arg', default=None,
+                       help=("Additional arguements consumed by "
```
Comment (miguelgrinberg) [inline flask_migrate/__init__.py]: Typo in the help message: arguements --> arguments

**Comment by miguelgrinberg** [comment]
This looks good, useful addition.

After you fix the typos, could you collapse all the commits into a single one?

Thanks!

**Comment by jessepavlis** [comment]
Typos are fixed, and the commits have been squashed.

**Pull request merged**